src/renderer/store/modules/delegate.js

```diff
@@ -25,6 +25,20 @@ export default {
       }
 
       return state.delegates[network.id][address]
+    },
+
+    byPublicKey: (state, _, __, rootGetters) => publicKey => {
+      const network = rootGetters['session/network']
+
+      if (!network || !state.delegates[network.id]) {
+        return false
+      }
+
+      const delegates = Object.values(state.delegates[network.id]).filter(delegate => {
```

Comment:
Could possibly just do `find` here (I returned false on failure in case you specifically want to return false on failure):

```
return Object.values(state.delegates[network.id]).find(delegate => {
  return delegate.publicKey === publicKey
}) || false
```